usaspending_api/etl/management/load_base.py

```diff
@@ -85,14 +84,15 @@ def handle(self, *args, **options):
             logger.info('Updating model description fields...')
             update_model_description_fields()
 
-        logger.info('Updating awards to reflect their latest associated transaction info...')
-        update_awards(tuple(award_update_id_list))
```

Comment:
We need this method to make sure an award's latest_transaction is correct and to make sure its totals (federal action obligation) is correct. 

 This code becomes even more valuable because of additional transactions being created by the nightly load.